hunts/src/puzzlesSlice.js

```diff
@@ -243,10 +244,23 @@ export const selectAllTags = createSelector(
           : tagNames.add(tag.name) && [...uniqueTags, tag],
```

Comment:
uniqueTags is of Tags objects, not tag names, which is needed for the current UI to display if a puzzle already has a tag. This is what the comment I added was getting at: when a default tag is used for the first time, a Tag is created in the DB and the version of the object we use in uniqueTags need to be the one created by the DB (with an ID), instead of the dummy one created in the constants file.

Reply:
Thanks for the elaboration. It'd be helpful to include this explanation in the comment on line 234 as well -- should make it easier for future reviewers and maintainers.